README.md

```diff
@@ -114,45 +99,95 @@ Curious how this works, under the hood? [__Read the Medium post__](https://mediu
-    <li className="col">Column 1</li>
-    <li className="col">Column 2</li>
-  </FlipMove>
+  <div>Hello</div>
+  <div>World</div>
 </div>
 ```
 
-FlipMove passes the `className` and `style` props along to the `ul` that needs to be created. Here's how it renders:
+Any unrecognized props to `<FlipMove>` will be delegated to this wrapper element:
 
-```html
-<div>
-  <ul class="row" style="background-color: red">
-    <li class="col">Column 1</li>
-    <li class="col">Column 2</li>
-  </ul>
+```jsx
+// JSX
+<FlipMove className="flip-wrapper" style={{ color: 'red' }}>
+  <div key="a">Hello</div>
+  <div key="b">World</div>
+</FlipMove>
+
+// HTML
+<div class="flip-wrapper" style="color: red;">
+  <div key="a">Hello</div>
+  <div key="b">World</div>
 </div>
 ```
 
-This works for all HTML props - there's no validation.
+You can supply a different element type with the `typeName` prop:
 
-You can pass `false` to `typeName` and a valid HTMLElement reference as `anchor` to render the children without a wrapping DOM node. Note that you will have to use react 16.0.0 or higher, and that all other props (such as `className` or `style`) will have no effect.
-```html
-<div ref={node => this.myCustomParent = node}>
-  <FlipMove typeName={false} anchor={this.myCustomParent}>
-    <li className="col">Column 1</li>
-    <li className="col">Column 2</li>
+```jsx
+// JSX
+<FlipMove typeName="ul">
+  <li key="a">Hello</li>
+  <li key="b">World</li>
+</FlipMove>
+
+// HTML
+<ul style="color: red;">
+  <li key="a">Hello</li>
+  <li key="b">World</li>
+</ul>
+```
+
+Finally, if you're using React 16 or higher, and Flip Move 2.10 or higher, you can use the new "wrapperless" mode. This takes advantage of a React Fiber feature, which allows us to omit this wrapping element:
+
+```jsx
+// JSX
+<div className="your-own-element">
+  <FlipMove typeName={null}>
+    <div key="a">Hello</div>
+    <div key="b">World</div>
   </FlipMove>
 </div>
+
+// HTML
+<div class="your-own-element">
+  <div key="a">Hello</div>
+  <div key="b">World</div>
+</div>
 ```
 
-If no anchor is provided, react-flip-move will try to use the components parent node. This, however, is a costly operation that violates component separation principles and thus is [highly discouraged](https://reactjs.org/docs/react-dom.html#finddomnode).
+Wrapperless mode is nice, because it makes FlipMove more "invisible", and makes it easier to integrate with parent-child CSS properties like flexbox. However, there are some things to note:
+
+- This is a new feature in FlipMove, and isn't as battle-tested as the traditional method. Please test thoroughly before using in production, and report any bugs!
+- FlipMove does some positioning magic for enter/exit animations - specifically, it temporarily applies `position: absolute` to its children. For this to work correctly, you'll need to make sure that `<FlipMove>` is within a container that has a non-static position (eg. `position: relative`), and no padding:
```

Comment:
since we already have the dom node, those styles could be overwritten after `findDOMNode`:
```js
  findDOMContainer = () => {
    // eslint-disable-next-line react/no-find-dom-node
    const domNode = ReactDOM.findDOMNode(this);
    const parentNode = domNode && domNode.parentNode;

    // This ought to be impossible, but handling it for Flow's sake.
    if (!parentNode || !(parentNode instanceof HTMLElement)) {
      return;
    }

    // Make sure parentNode has position relative and no padding
    parentNode.style.position = 'relative';
    parentNode.style.padding = 0;

    this.parentData.domNode = parentNode;
  };
```

Reply:
```
parentNode.style.position = 'relative';
```

We can do that only if `window.getComputedStyle(parentNode) === 'static')`
And we can't change the padding as it would break the layout

Reply:
Yeah, so I thought about applying a "default" relative position if it isn't set to something other than `static`, but it feels a bit like it crosses a line; we'd be mutating nodes outside of Flip Move's control. It could cause confusion for users if they have absolutely-positioned things down-tree.

So either we apply position: relative, which fixes FlipMove but breaks their layout, or we do nothing, and it breaks enter/exit animations. The latter is clearly a FlipMove issue, and so they'll come here and we can advise them... but the former might appear to them as a CSS issue / they won't understand why things are broken, why a random `position: relative` is being applied to their `div`...

Maybe we can apply relative positioning if it's static, and also `console.warn` to let them know what's happening? But, if the user doesn't need enter/leave animations, they don't need this `position: relative`, and so the warning would just be console spam.

That said, it's really rare that `position: static` is actually desirable (I've seen a trend where people just add a `* { position: relative }` global CSS rule), so I think it's probably best to optimize for the common usecase.

Gonna do it in a subsequent PR to make reviews easier, but I'll apply @Hypnosphi's static check, and issue a console.warn when the change is necessary.